proxy/http2/Http2Stream.cc

```diff
@@ -603,9 +603,8 @@ Http2Stream::update_write_request(IOBufferReader *buf_reader, int64_t write_len,
     }
   }
 
-  Http2StreamDebug("write_vio.nbytes=%" PRId64 ", write_vio.ndone=%" PRId64 ", write_vio.write_avail=%" PRId64
-                   ", reader.read_avail=%" PRId64,
-                   write_vio.nbytes, write_vio.ndone, write_vio.get_writer()->write_avail(), bytes_avail);
+  Http2StreamDebug("write_vio.nbytes=%" PRId64 ", write_vio.ndone=%" PRId64 ", reader.read_avail=%" PRId64, write_vio.nbytes,
+                   write_vio.ndone, bytes_avail);
```

Comment:
This change is definitely good. We should not call functions in Debug that have a side effect.